scripts/menu/client/cl_menu.lua

```diff
@@ -235,6 +235,7 @@ local function toggleFreecam(enabled)
   
   if enabled then
     freecamVeh = GetVehiclePedIsIn(ped, false)
+    PedIsIn(ped, false)
```

Comment:
lol... did you even test before committing this?

Reply:
i didnt even change that stuff wtf.